merlin/examples/generator.py

```diff
@@ -74,25 +73,31 @@ def write_example(src_path, dst_path):
 
 def list_examples():
     """List all available examples."""
-    examples = gather_examples()
+    examples = gather_example_dirs()
 
     headers = ["name", "description"]
     rows = []
-    for example in examples:
-        with open(
-            os.path.join(os.path.join(EXAMPLE_DIR, example), example + ".yaml")
-        ) as f:
-            example_descrips = yaml.safe_load(f)["description"]
-        rows.append([example_descrips["name"], example_descrips["description"]])
+    for example_dir in examples:
```

Comment:
i feel like this could be shorted to for spec in gather_all_examples()

Reply:
Yep.